Tasks/SqlAzureDacpacDeployment/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 11
+    "Patch": 12
```

Comment:
Can you run gulp test and let us know if it succeeds. That is our L0 and also ensure formatting of files are not impacted. I can merge this once you confirm the same.